components/collective-navbar/ActionsMenu.js

```diff
@@ -233,7 +233,7 @@ const CollectiveNavbarActionsMenu = ({ collective, callsToAction, hiddenActionFo
                     )}
                     {callsToAction.hasDashboard && !hasNewAdminPanel && (
                       <MenuItem isHiddenOnMobile={hiddenActionForNonMobile === NAVBAR_ACTION_TYPE.DASHBOARD}>
-                        <StyledLink as={Link} href={`/${collective.slug}/dashboard`}>
+                        <StyledLink as={Link} href={`/${getCollectivePageRoute(collective)}/dashboard`}>
```

Comment:
:no_entry_sign: There's no such route as `/parent/collective/dashboard` (and it wouldn't make much sense to have one)

Reply:
Ops, this is a mistake; I've reverted this part.